spring-5-reactive-client/src/main/java/com/baeldung/reactive/service/EmployeeService.java

```diff
@@ -0,0 +1,55 @@
+package com.baeldung.reactive.service;
+import com.baeldung.reactive.model.Employee;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+
+public class EmployeeService {
+
+    private WebClient webClient;
+    public static String PATH_PARAM_BY_ID = "/employee/{id}";
+    public static String ADD_EMPLOYEE = "/employee";
+
+    public EmployeeService(WebClient _webClient) {
+        this.webClient = _webClient;
```

Comment:
This `_` convention is not used at Baeldung. Let's replace it with the more conventional `this.webClient = webClient`.